src/patternfly/components/Tabs/tabs.scss

```diff
@@ -1,104 +1,118 @@
-  --pf-c-tabs__item--m-current--BorderTopWidth: var(--pf-global--BorderWidth--md);
-
-  // Secondary tab items
-  --pf-c-tabs__item--hover--Color: var(--pf-global--Color--dark-200);
 
   // Tab buttons
-  --pf-c-tabs__button--Color: var(--pf-global--Color--100);
-  --pf-c-tabs__button--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-tabs__button--Color: var(--pf-global--Color--200);
   --pf-c-tabs__button--Background: transparent;
   --pf-c-tabs__button--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
   --pf-c-tabs__button--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-tabs__button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-tabs__button--PaddingBottom: var(--pf-global--spacer--sm);
-  --pf-c-tabs__button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--before--BorderStyle: solid;
+  --pf-c-tabs__item--m-current--c-tabs__button--Color: var(--pf-global--Color--100);
+  --pf-c-tabs__item--m-current--c-tabs__button--after--BorderColor: var(--pf-global--active-color--100);
+
+  // Roomy
+  --pf-c-tabs--m-roomy--c-tabs__button--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-tabs--m-roomy--c-tabs__button--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-tabs--m-roomy--c-tabs__button--md--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-tabs--m-roomy--c-tabs__button--md--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Button before
+  --pf-c-tabs__button--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-tabs__button--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+
+  // Button after
+  --pf-c-tabs__button--after--BorderColor: transparent;
+  --pf-c-tabs__button--after--BorderWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-tabs__button--hover--after--BorderColor: var(--pf-global--BorderColor--light-100);
+  --pf-c-tabs__button--child--MarginRight: var(--pf-global--spacer--md);
 
   // Scroll buttons
-  --pf-c-tabs__scroll-button--Width: var(--pf-global--spacer--xl);
-  --pf-c-tabs__scroll-button--ZIndex: var(--pf-global--ZIndex--xs);
-  --pf-c-tabs__scroll-button--m-secondary--hover--Color: var(--pf-global--active-color--100);
-  --pf-c-tabs__scroll-button--m-secondary--hover--Color: var(--pf-global--active-color--100);
-  --pf-c-tabs__scroll-button--m-secondary-right--m-start-current--Color: var(--pf-global--active-color--100);
-  --pf-c-tabs__scroll-button--m-secondary--nth-of-type-1--BoxShadow: var(--pf-global--BoxShadow--lg-right);
-  --pf-c-tabs__scroll-button--m-secondary--nth-of-type-2--BoxShadow: var(--pf-global--BoxShadow--lg-left);
+  --pf-c-tabs__scroll-button--Width: var(--pf-global--spacer--2xl);
```

Comment:
I'm not sure why we defined a width here initially. Do you think it would make more sense to define the button width by padding?